.cliversion

```diff
@@ -0,0 +1 @@
+1.0.0-preview3-003708
```

Comment:
I'm not sure where I got that from, it looks like the actual rel-1.0.0 version that we are fetching with the old script is 1.0.0-preview3-003786.  Updated.